packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -929,13 +930,16 @@ async function processNetworkData(
 
   // Filter by RequestInfo (because they have execution points)
   // then map RequestInfo to RequestEventInfo using ids
-  const [beginIndex, endIndex] = findSliceIndices(ids, begin.point, end.point, (id, point) => {
```

Comment:
Was the utility broken or what? It isn't clear from this PR (empty description) why deleting that code and its tests were worth doing. Seems like the "fix" was the changes to the e2e test expectations.

Reply:
Sorry, forgot to add the description - yeah, it was broken. The test expectations needed to be updated because the Cypress panel now shows 3 network events for that test.